test/e2e/e2e_suite_test.go

```diff
@@ -174,3 +187,21 @@ func getMCPUpdateValueFromEnv(envVar string, fallback time.Duration) time.Durati
 	Expect(err).NotTo(HaveOccurred())
 	return timeout
 }
+
+func pauseMCPs(nodeGroups []nropv1alpha1.NodeGroup, pause bool) {
+	mcps, err := controllers.GetNodeGroupsMCPs(context.TODO(), e2eclient.Client, nodeGroups)
+	if err != nil {
+		klog.Warningf("failed to get the MCPs resources: %v", err)
+		return
+	}
+
+	if len(mcps) == 0 {
+		klog.Warningf("no MCPs resources found", err)
+	}
+
+	for i := range mcps {
+		mcps[i].Spec.Paused = pause
+		err = e2eclient.Client.Update(context.TODO(), mcps[i])
```

Comment:
@fromanirh is there a way to update all the MCPs together, using ```machineConfigList```?

Reply:
I actually don't know